crates/net/eth-wire-types/src/snap.rs

```diff
@@ -0,0 +1,177 @@
+use alloy_primitives::{Bytes, B256};
```

Comment:
```suggestion
use alloy_primitives::{Bytes, B256};
use alloc::vec::Vec;
```
default to making no_std compatible wherever possible from start